the-blue-alliance-ios/View Elements/Events/Stats/EventStatsConfigurator2016.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+import UIKit
+
+class EventStatsConfigurator2016: EventStatsConfigurator {
+
+    static func configureDataSource(_ snapshot: inout NSDiffableDataSourceSnapshot<String, InsightRow>, _ qual: [String : Any]?, _ playoff: [String : Any]?) {
+        // Match Stats
+        var matchStats: [InsightRow] = []
+
+        matchStats.append(InsightRow(title: "High Score", qual: highScoreString(qual, "high_score"), playoff: highScoreString(playoff, "high_score")))
```

Comment:
Let's see if we can DRY these... part of the concern is that we mismatch a key and show the wrong info